rust/datafusion/src/optimizer/filter_push_down.rs

```diff
@@ -370,10 +368,9 @@ mod tests {
         let plan = LogicalPlanBuilder::from(&table_scan)
             .aggregate(
                 vec![col("a")],
-                vec![aggregate_expr("SUM", col("b"), DataType::Int32)
-                    .alias("total_salary")],
+                vec![aggregate_expr("SUM", col("b")).alias("total_salary")],
             )?
-            .filter(col("a").gt(Expr::Literal(ScalarValue::Int64(10))))?
```

Comment:
this was a simplification, so that the tests do not need to use `DataType` nor `ScalarValue`.